### PR TITLE
hardening/ErrorPutAttributes

### PR DESCRIPTION
**Pull request by rbl219**
Resolved issue  #1320
Modifying error Not  Found

### CODE REVIEW

File: src/lib/serviceRoutinesV2/putEntity.cpp [hunk 1]
```diff
@@ -93,9 +93,12 @@ std::string putEntity
   {
```
Comment (fgalan) [inline src/lib/serviceRoutinesV2/putEntity.cpp]: CNR entry missing? Or it was added in a previous PR?

Comment (rbl219) [inline src/lib/serviceRoutinesV2/putEntity.cpp]: Yes sorry, I forgot add CNR line

Comment (rbl219) [inline src/lib/serviceRoutinesV2/putEntity.cpp]: Fix on commit e4ab14a

**Comment by fgalan** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**